cpp/src/gandiva/string_function_holder_test.cc

```diff
@@ -212,6 +213,189 @@ TEST_F(TestLikeHolder, TestMultipleEscapeChar) {
   auto status = LikeHolder::Make("ab\\_", "\\\\", &like_holder);
   EXPECT_EQ(status.ok(), false) << status.message();
 }
+
+// Tests related to the REGEXP_EXTRACT function
+class TestExtractHolder : public ::testing::Test {
+ protected:
+  ExecutionContext execution_context_;
+};
+
+TEST_F(TestExtractHolder, TestSimpleExtract) {
```

Comment:
Can you also add integration test in projector_tests?

Reply:
I added the test, I found an error with the implementation when needs to match the string partially. I will look at it, I think it is a wrong use of the RE2 library